<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>631</width>
    <height>558</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QToolBox" name="mainLayoutToolbox">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <widget class="QWidget" name="dataLoadingPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>476</width>
         <height>466</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Data Loading</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_9">
        <item row="0" column="0">
         <layout class="QGridLayout" name="dataLoadingLayout">
          <item row="0" column="1">
           <layout class="QVBoxLayout" name="dataset2Layout">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="dataset2Label">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>15</height>
               </size>
              </property>
              <property name="text">
               <string>Dataset B</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="dataset2FileLoad">
              <property name="text">
               <string>Select File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="inputParamsButton2">
              <property name="text">
               <string>Input Params</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListView" name="dataset2Columns">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="dragDropOverwriteMode">
               <bool>true</bool>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::DropOnly</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="3">
           <layout class="QVBoxLayout" name="dataButtonLayout">
            <item>
             <widget class="QPushButton" name="addOneButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="addAllButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>&gt;&gt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="removeOneButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>&lt;</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="removeAllButton">
              <property name="enabled">
               <bool>false</bool>
              </property>
              <property name="maximumSize">
               <size>
                <width>40</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>&lt;&lt;</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="5">
           <layout class="QVBoxLayout" name="comparisonColumnsLayout">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="comparisonColumnsLabel">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>15</height>
               </size>
              </property>
              <property name="text">
               <string>Comparisons</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QTableView" name="comparisonColumnsTable">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>100</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="alternatingRowColors">
               <bool>true</bool>
              </property>
              <property name="selectionMode">
               <enum>QAbstractItemView::SingleSelection</enum>
              </property>
              <property name="selectionBehavior">
               <enum>QAbstractItemView::SelectRows</enum>
              </property>
              <attribute name="horizontalHeaderCascadingSectionResizes">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderMinimumSectionSize">
               <number>30</number>
              </attribute>
              <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
               <bool>true</bool>
              </attribute>
              <attribute name="horizontalHeaderStretchLastSection">
               <bool>false</bool>
              </attribute>
              <attribute name="verticalHeaderStretchLastSection">
               <bool>false</bool>
              </attribute>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="0">
           <layout class="QVBoxLayout" name="dataset1Layout">
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="dataset1Label">
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>15</height>
               </size>
              </property>
              <property name="text">
               <string>Dataset A</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="dataset1FileLoad">
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="text">
               <string>Select File</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="inputParamsButton1">
              <property name="text">
               <string>Input Params</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListView" name="dataset1Columns">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>150</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="dragDropOverwriteMode">
               <bool>true</bool>
              </property>
              <property name="dragDropMode">
               <enum>QAbstractItemView::DropOnly</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item row="0" column="2">
           <widget class="Line" name="addremoveLineLeft">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
          <item row="0" column="4">
           <widget class="Line" name="addremoveLineRight">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="dataDetailPage">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>476</width>
         <height>466</height>
        </rect>
       </property>
       <attribute name="label">
        <string>Data Detail</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_8">
        <property name="leftMargin">
         <number>0</number>
        </property>
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item row="0" column="0">
         <widget class="QFrame" name="dataDetailLayout">
          <layout class="QGridLayout" name="gridLayout_3">
           <item row="0" column="1">
            <widget class="QTabWidget" name="comparisonsTabLayout">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="tabShape">
              <enum>QTabWidget::Triangular</enum>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="setupTab">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="maximumSize">
               <size>
                <width>16777215</width>
                <height>16777215</height>
               </size>
              </property>
              <attribute name="title">
               <string>Setup</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout">
               <item row="0" column="0">
                <layout class="QGridLayout" name="gridLayout_10">
                 <item row="0" column="0">
                  <layout class="QVBoxLayout" name="setupInputsLayout">
                   <item>
                    <widget class="QLabel" name="comparisonSetupLabel">
                     <property name="text">
                      <string>Comparison Setup:</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QComboBox" name="comparisonsComboBox"/>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="addDiffCheckbox">
                     <property name="enabled">
                      <bool>true</bool>
                     </property>
                     <property name="text">
                      <string>Add Difference Column</string>
                     </property>
                     <property name="checked">
                      <bool>true</bool>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="performValidationsCheckbox">
                     <property name="text">
                      <string>Perform Validations</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QCheckBox" name="createVizCheckbox">
                     <property name="text">
                      <string>Create Visualizations</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <layout class="QGridLayout" name="gridLayout_7">
                     <item row="0" column="0" alignment="Qt::AlignHCenter">
                      <widget class="QLabel" name="validationConfiguration">
                       <property name="text">
                        <string>Validation Configuration:</string>
                       </property>
                      </widget>
                     </item>
                     <item row="1" column="0">
                      <widget class="QPushButton" name="openConfigButton">
                       <property name="text">
                        <string>Open Config File</string>
                       </property>
                      </widget>
                     </item>
                     <item row="2" column="0">
                      <widget class="QTableView" name="configTable">
                       <property name="sizeAdjustPolicy">
                        <enum>QAbstractScrollArea::AdjustIgnored</enum>
                       </property>
                       <property name="selectionMode">
                        <enum>QAbstractItemView::SingleSelection</enum>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
               <item row="1" column="0">
                <layout class="QHBoxLayout" name="finalButtonsLayout">
                 <item>
                  <widget class="QPushButton" name="compareButton">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Compare</string>
                   </property>
                  </widget>
                 </item>
                 <item>
                  <widget class="QPushButton" name="resetButton">
                   <property name="enabled">
                    <bool>false</bool>
                   </property>
                   <property name="text">
                    <string>Reset</string>
                   </property>
                  </widget>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="comparisonTab">
              <attribute name="title">
               <string>Comparisons</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_4">
               <item row="0" column="0">
                <layout class="QHBoxLayout" name="horizontalLayout_2">
                 <item>
                  <layout class="QVBoxLayout" name="comparisonLayout">
                   <item alignment="Qt::AlignHCenter">
                    <widget class="QLabel" name="comparisonLabel">
                     <property name="text">
                      <string>Comparison</string>
                     </property>
                    </widget>
                   </item>
                   <item>
                    <widget class="QTableView" name="comparisonTable">
                     <property name="alternatingRowColors">
                      <bool>true</bool>
                     </property>
                     <attribute name="horizontalHeaderCascadingSectionResizes">
                      <bool>true</bool>
                     </attribute>
                    </widget>
                   </item>
                   <item>
                    <layout class="QVBoxLayout" name="validationLayout"/>
                   </item>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="chartsTab">
              <attribute name="title">
               <string>Plots</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_11">
               <item row="0" column="0">
                <layout class="QVBoxLayout" name="verticalLayout">
                 <item>
                  <layout class="QGridLayout" name="plotsGridLayout">
                   <property name="leftMargin">
                    <number>0</number>
                   </property>
                   <property name="rightMargin">
                    <number>0</number>
                   </property>
                   <property name="spacing">
                    <number>3</number>
                   </property>
                  </layout>
                 </item>
                </layout>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QTabWidget" name="datasetDetailsTabLayout">
             <property name="tabShape">
              <enum>QTabWidget::Triangular</enum>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="dataset1Tab">
              <attribute name="title">
               <string>Dataset 1</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_6">
               <item row="0" column="0">
                <widget class="QPushButton" name="datasetDetails1Button">
                 <property name="text">
                  <string>Dataset Details</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QTableView" name="dataframe1Table">
                 <property name="alternatingRowColors">
                  <bool>true</bool>
                 </property>
                 <property name="sortingEnabled">
                  <bool>true</bool>
                 </property>
                 <attribute name="horizontalHeaderMinimumSectionSize">
                  <number>60</number>
                 </attribute>
                 <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
                  <bool>false</bool>
                 </attribute>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="dataset2Tab">
              <attribute name="title">
               <string>Dataset 2</string>
              </attribute>
              <layout class="QGridLayout" name="gridLayout_5">
               <item row="0" column="0">
                <widget class="QPushButton" name="datasetDetails2Button">
                 <property name="text">
                  <string>Dataset Details</string>
                 </property>
                </widget>
               </item>
               <item row="1" column="0">
                <widget class="QTableView" name="dataframe2Table">
                 <property name="alternatingRowColors">
                  <bool>true</bool>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="loggingBoxLayout">
      <property name="leftMargin">
       <number>10</number>
      </property>
      <property name="topMargin">
       <number>10</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>10</number>
      </property>
      <item row="0" column="0">
       <widget class="QPlainTextEdit" name="loggingBox">
        <property name="minimumSize">
         <size>
          <width>100</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>200</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
